2024-10-31-introducing-the-catalyst-project-community-partner-highlights.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: Introducing the Catalyst Project Community Partner Highlights
+authors: 
+- Katie Pratt
+- slldec
+image: https://media.licdn.com/dms/image/v2/D5610AQGPZaXMJKu5Wg/image-shrink_800/image-shrink_800/0/1730385031498?e=1731063600&v=beta&t=-F0ZI4Zj-oKi2cRQyrZcypICq2RuwQAaNyJQubPH3mg
+photos:
+  name: Picture by The Catalyst Project
+  license: CC BY-SA 4.0
+  url: https://unsplash.com/@thisisengineering
```

Comment:
https://www.linkedin.com/posts/2i2c-org_as-part-of-our-work-on-the-catalyst-project-activity-7257760964647755777-4fNn